app/docker/components/network-macvlan-form/networkMacvlanForm.html

```diff
@@ -0,0 +1,97 @@
+<div>
+  <!-- selector -->
+  <div class="col-sm-12">
+    <div class="form-group" style="margin-bottom: 0">
+      <div class="boxselector_wrapper">
```

Comment:
If there is no availableNetworks, I think that we should not show the "I want to configure a network before deploying it".

UX wise, it does not make sense to show a box selector if there is only choice available. Instead, a small information message should be displayed.

Reply:
Hiding the box selectors in one context and displaying them in another isn't UX friendly, as the view isn't the same depending on the same "above" context (means, you can't remove things from the screen, for a same visible context, based on things not shown on the screen).

Maybe we should keep the 2 selectors visible but disable the "Creation" one if the user doesn't have config networks (turning it grey + small information message) ?

Reply:
Agree, I like the view you posted.